source/PreferencesPanel.cpp

```diff
@@ -296,7 +296,7 @@ void PreferencesPanel::DrawControls()
 	const Color &bright = *GameData::Colors().Get("bright");
 	
 	// Check for conflicts.
-	Color red(.3f, 0.f, 0.f, .3f);
+	Color warning(.5f, .3f, 0.f, 0.f);
```

Comment:
This is a pretty harsh yellow compared to the duller red we had before. 
![image](https://user-images.githubusercontent.com/17688683/106563114-29b34b00-64e0-11eb-8d0c-8b1382edd836.png)
![image](https://user-images.githubusercontent.com/17688683/106563151-38016700-64e0-11eb-9611-4683ce0f2807.png)
How about (.315f .27f .12f 1.f) instead?
![image](https://user-images.githubusercontent.com/17688683/106564582-42bcfb80-64e2-11eb-8789-d7bc178dbf3b.png)
And while we're here, how about defining this color in interfaces.txt like the other colors above? (Speaking of the interface file, the suggested color is actually a 50% brighter version of the "warning back" color already defined there.)